docker/postgres/auth-schema.sql

```diff
@@ -88,4 +88,7 @@ $$ language sql stable;
 GRANT ALL PRIVILEGES ON SCHEMA auth TO postgres;
 GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA auth TO postgres;
 GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA auth TO postgres;
-ALTER ROLE postgres SET search_path = "$user", public, auth;
\ No newline at end of file
+ALTER ROLE postgres SET search_path = "$user", public, auth;
+
+GRANT USAGE ON SCHEMA auth TO authenticated;
+GRANT EXECUTE ON ALL FUNCTIONS IN SCHEMA auth TO authenticated;
```

Comment:
Thanks for jumping in to help @jasonho-lynx.

I think just granting usage on the schema should be enough for having access to the functions as these are not revoked from `public`.

So this should be enough:

```suggestion
GRANT USAGE ON SCHEMA auth TO authenticated;
```

Reply:
Also, maybe the `anon` and `service_role` roles should also have access to the `auth` functions.

```sql
GRANT USAGE ON SCHEMA auth TO anon, authenticated, service_role;
```

In the case of anon, the `auth.uid()` function and the like would always give null, but that might be useful for some RLS logic.

Reply:
Okay, updated!